README.md

```diff
@@ -1 +1,6 @@
 # frames
+
+/simple
```

Comment:
Maybe we should provide more detailed readme file?

Reply:
yes readme I will add after this.